main.go

```diff
@@ -81,13 +82,17 @@ func main() {
 	}
 
 	config := ctrl.GetConfigOrDie()
+
 	mgr, err := ctrl.NewManager(config, ctrl.Options{
 		Scheme:             scheme,
 		MetricsBindAddress: metricsAddr,
+		WebhookServer: &webhook.Server{
+			Port:          9443,
+			TLSMinVersion: "1.2",
+		},
 		NewCache: cache.MultiNamespacedCacheBuilder(
 			[]string{controllers.ComponentNamespace, provisioning.OpenshiftConfigNamespace}),
 		LeaderElection: enableLeaderElection,
-		Port:           9443,
 		CertDir:        "/etc/cluster-baremetal-operator/tls",
```

Comment:
This has to be moved to the WebhookServer as well.

Reply:
Ack, I will update things